event.c

```diff
@@ -1377,11 +1377,13 @@ event_signal_closure(struct event_base *base, struct event *ev)
 {
 #if defined(__clang__)
 #elif defined(__GNUC__)
+#if __GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 6)
```

Comment:
Please, submit changes for GCC<4 separately.

Reply:
Sorry! I only want to submit the first two commits (the code of evhttp_serve). But I don’t know why the subsequent commit was also pulled (I am using libevent on an old chip. This part of the code is to adapt to the old chip but it was pulled by mistake).

Reply:
It happened because you made a PR from your **master** branch instead of from a dedicated branch.

Reply:
> It happened because you made a PR from your **master** branch instead of from a dedicated branch.

I see. Thank you for your answer. Obviously I am still very new to multi-person collaboration git projects, but I will try my best to continue to improve.